.github/pull.yml

```diff
@@ -0,0 +1,11 @@
+version: "1"
+rules:
+  - base: stable
+    upstream: ember-cli:stable
```

Comment:
this has no effect on this branch or the target branch atm, as I'm tracking the ember-cli "stable" branch, but this is what was written at the time over on https://github.com/NullVoxPopuli/ember-new-output/commit/043f9eadc2cf9fe597a15497da585cc2559f928d